websites/M/Maliki/dist/metadata.json

```diff
@@ -0,0 +1,40 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "RisingSunLight",
+		"id": "240521747852558347"
+	},
+	"service": "Maliki",
+	"description": {
+		"en": "Presence which helps you to follow your activity on the official website of Maliki !",
+		"fr": "Présence qui permet de suivre votre activité sur le site officiel de Maliki !"
+	},
+	"url": [
+		"maliki.com",
+		"malimode.maliki.com"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/QDOaEbp.jpg",
+	"thumbnail": "https://i.imgur.com/LDOrHM3.jpg",
```

Comment:
if you can make the resolution of this one a bit better that would be nice

Reply:
I've made it better, idk if it's ok, but it's the best I can do 5340f44d9f71338d887645989a3c948545bcb49b